tgui/packages/tgui/interfaces/GhostHudPanel.js

```diff
@@ -12,11 +12,12 @@ export const GhostHudPanel = (props, context) => {
   } = data;
   return (
     <Window resizable theme="nologo">
```

Comment:
This probably doesnt need to be `resizable`

Reply:
No, it does not, will remove.